docs/CHANGELOG.asciidoc

```diff
@@ -45,6 +46,236 @@ This section summarizes the changes in each release.
 //[float]
 //=== Known Issues
 
+
+[[release-notes-7.0.0-alpha2]]
+== {kib} 7.0.0-alpha2
+
+coming[7.0.0-alpha2]
+
+[float]
+[[breaking-7.0.0-alpha2]]
+=== Breaking changes
+
+For more details about breaking changes in this release, see 
+<<breaking-changes-7.0, Breaking changes in 7.0>>.
+
+[float]
+[[enhancement-7.0.0-alpha2]]
+=== Enhancements
+Canvas::
+* Implements a clipboard and stores the copied elements in `localStorage` {pull}25890[#25890]
+* Adds the ability to reuse assets without editing an element's expression {pull}25764[#25764]
+* Adds the `clear` function {pull}26397[#26397]
+* Adds a template tab to the workpad manager where users can find workpad templates, 
+demos, and tutorials to help them get started {pull}23966[#23966]
+Design::
+* Updates logos for marketing {pull}25489[#25489]
+* Updates to EUI 5.3.0 {pull}26485[#26485]
+* Adds Kibana 7.0 breadcrumbs to home screen {pull}26605[#26605]
+* Adds EUI 5.6.0 {pull}26839[#26839]
+* Moves elastic/eui typings to single file {pull}23950[#23950]
+Index Lifecycle Management::
+* Implements a CRUD UI for index lifecycle management policies {pull}25553[#25553]
```

Comment:
I think index lifecycle management should be capitalized - it's the name of a feature

Reply:
Thanks, @rayafratkina - I made the two suggested changes.